db/python/layers/web.py

```diff
@@ -459,7 +459,7 @@ async def get_project_summary(
             'sequencing_technology',
             'sequencing_type',
         }
-        ignore_sg_meta_keys: set[str] = set()
+        ignore_sg_meta_keys: set[str] = {'facility'}
```

Comment:
Any particular reason for this?

Reply:
Facility appears on both assays and sequencing groups, figured it made sense to omit from SGs and keep on assays in case any assays in the same SG came from different facilities?

Reply:
I think this is a byproduct of the test data, and not of real data (I could be mistaken though)

Reply:
Ah ok - I'll add it back in then (we can always customise this later)